save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/index/IndexLogoButtons.kt

```diff
@@ -0,0 +1,88 @@
+/**
+ * Beautiful logos with effects for SAVE and VULN services
+ */
+
+package com.saveourtool.save.frontend.components.views.index
+
+import com.saveourtool.save.validation.FrontendRoutes
+import js.core.jso
+import react.ChildrenBuilder
+import react.VFC
+import react.dom.html.ReactHTML
+import react.dom.html.ReactHTML.div
+import web.cssom.Border
+import web.cssom.ClassName
+import web.cssom.rem
+
+val logoButtons = VFC {
+    div {
+        className = ClassName("row logo-parent mb-5")
+        div {
+            className = ClassName("col-3 text-center")
+        }
+
+        div {
+            className = ClassName("col-3 text-center logo-main")
+            creationCard(
+                "#/${FrontendRoutes.SAVE}",
+                "img/save-logo-bg.jpg",
+            )
+            neonLightingText("SAVE", "#/${FrontendRoutes.SAVE}")
+        }
+
+        div {
+            className = ClassName("col-3 text-center logo-main")
+            creationCard(
+                "#/${FrontendRoutes.VULNERABILITIES}",
+                "img/vuln-logo-bg.jpg",
+            )
+
+            neonLightingText("VULN", "#/${FrontendRoutes.VULNERABILITIES}")
+        }
+
+        div {
+            className = ClassName("col-3 text-center")
+        }
+    }
+}
+
+private fun ChildrenBuilder.creationCard(url: String, img: String) {
+    ReactHTML.div {
```

Comment:
```suggestion
    div {
```

And so on in this file

Reply:
stupid IDEA